### ISSUE TITLE
Fix critical security issue with proxy-agent => vm2

### ISSUE THREAD

**Issue by cristan** [motdotla/node-lambda#695]
I added "node-lambda": "^1.2.0", (the [latest npm version](https://www.npmjs.com/package/node-lambda)) to my project.

This has caused me to have version 3.9.17 from vm2, which has these 2 vulnerabilities:
* vm2 Sandbox Escape vulnerability - https://github.com/advisories/GHSA-cchq-frgv-rjh5
* vm2 Sandbox Escape vulnerability - https://github.com/advisories/GHSA-g644-9gfx-q4q4

If I do an `npm why vm2`, I end up at this project:

```
~npm why vm2
vm2@3.9.19
node_modules/vm2
  vm2@"^3.9.17" from degenerator@3.0.4
  node_modules/degenerator
    degenerator@"^3.0.2" from pac-resolver@5.0.1
    node_modules/pac-resolver
      pac-resolver@"^5.0.0" from pac-proxy-agent@5.0.0
      node_modules/pac-proxy-agent
        pac-proxy-agent@"^5.0.0" from proxy-agent@5.0.0
        node_modules/proxy-agent
          proxy-agent@"^5.0.0" from node-lambda@1.2.0
          node_modules/node-lambda
            node-lambda@"^1.2.0" from the root project
```

At [master](https://github.com/motdotla/node-lambda/blob/master/package.json), you _are_ using a secure version of proxy-agent. Problem is, this has been [merged](https://github.com/motdotla/node-lambda/commit/75f153652980c09307bbf63cb198d50562d5ad39) by dependabot 6 months ago, while 1.2.0 was released a year ago.

So please release a new release, so users of this library won't have critical security vulnerabilities.

**Comment by DeviaVir** [comment]
Resolved with the bump to 1.2.1?

**Comment by cristan** [comment]
*doublechecks:

On 1.2.0:
```
vm2  *
Severity: critical
vm2 Sandbox Escape vulnerability - https://github.com/advisories/GHSA-cchq-frgv-rjh5
vm2 Sandbox Escape vulnerability - https://github.com/advisories/GHSA-g644-9gfx-q4q4
fix available via `npm audit fix`
node_modules/vm2
  degenerator  3.0.0 - 4.0.4
  Depends on vulnerable versions of vm2
  node_modules/degenerator
    pac-resolver  5.0.0 - 6.0.2
    Depends on vulnerable versions of degenerator
    node_modules/pac-resolver
      pac-proxy-agent  5.0.0 - 6.0.4
      Depends on vulnerable versions of pac-resolver
      node_modules/pac-proxy-agent
        proxy-agent  5.0.0 - 6.2.2
        Depends on vulnerable versions of pac-proxy-agent
        node_modules/proxy-agent
          node-lambda  0.20.0 - 1.2.0
          Depends on vulnerable versions of proxy-agent
          node_modules/node-lambda
```

1.2.1:
Nothing, it's gone.

Excellent! Thank y'all very much!
